.vale/styles/RedHat/DoNotUseTerms.yml

```diff
@@ -18,10 +18,12 @@ swap:
   fubar: 'Do not use "fubar". This term is an acronym of a profanity that is sometimes used by developers in code.'
   \bred\b(?!-? *hat)|orange|yellow|green|blue|indigo|violet: 'Do not use colors to describe something unless it is also described non-visually, for example, with the name of the item.'
   kerberize|kerberized: 'Do not use "kerberize" to refer to applications or services that use Kerberos authentication. Refer to such applications as "Kerberos-aware" or "Kerberos-enabled", or rewrite the sentence.'
+  MCP|Mini Control Program|mini control program: 'Do not use MCP or Mini Control Program to refer to the embedded operating system. Do not refer at all to the embedded operating system unless it is clearly technically required to either use or service an IBM product or service. If the context of the communication clearly requires a technical reference to the embedded operating system, use the term Linux kernel 2.6.x, where x is the modification level.'
```

Comment:
Not sure about "MCP" here, can we remove this term? https://www.anthropic.com/news/model-context-protocol is about to blow up in the world of AI, and we'll probably end up removing it in 6 monoths.